lib/topological_inventory/openshift/operations/core/retriever.rb

```diff
@@ -0,0 +1,27 @@
+require "topological_inventory-api-client"
+
+module TopologicalInventory
+  module Openshift
+    module Operations
+      module Core
+        class Retriever
+          def initialize(id)
+            @id = id
+            uri = URI.parse(ENV["TOPOLOGICAL_INVENTORY_URL"])
+            TopologicalInventoryApiClient.configure do |config|
+              config.base_path = "#{ENV["PATH_PREFIX"]}/topological-inventory/v0.0/"
```

Comment:
Prefer v0.1, I see v0.0 as deprecated.  We won't be adding anything new to it.

Reply:
:+1: